comp/languagedetection/client/client.go

```diff
@@ -287,7 +287,6 @@ func (c *client) retryProcessEventsWithoutPod(containerIDs []string) {
 			c.handleProcessEvent(procEvent, true)
 		}
 	}
-	c.processesWithoutPod = make(map[string]*eventsToRetry)
```

Comment:
We shouldn't clear `processesWithoutPod` after handling a Pod Set event, otherwise we will lose all waiting process events even if the pod set event has nothing to do with the process events.